lib/humanized_id.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module HumanizedId
+  DEFAULT_GENERATION_LENGTH = 24
+  CHARACTERSET =
+    (('0'..'9').to_a + ('A'..'Z').to_a - %w(0 1 5 8 A B E I L O S U)).freeze
```

Comment:
yeah, you could almost have
NUMBERS_LETTERS_SIMILAR = [0 O 1 I L 5 S 8 B]
VOWELS = [A E I O U]
Then CHARACTERSET = 0-9 + A-Z - VOWELS - NUMBERS_LETTERS_SIMILAR